Tests/FunctionalTests/Sources/FTRSwiftTests.swift

```diff
@@ -153,4 +159,24 @@ class FunctionalTestRigSwiftTests: XCTestCase {
 
     return GREYElementMatcherBlock.init(matchesBlock: matches, descriptionBlock: description)
   }
+
+  func registerForChanges() -> GREYActionBlock {
+    NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(textFieldChanged), name: UITextFieldTextDidChangeNotification, object: nil)
+    return GREYActionBlock.actionWithName("Register to editing events") {
+      (element: AnyObject!, _: UnsafeMutablePointer<NSError?>) -> Bool in
+      element.addTarget(self, action: #selector(self.editingChanged), forControlEvents: .EditingChanged)
+      return true
+    }
+
```

Comment:
Done. I think I opened the PR to changes by contributors. That should cut some back and forth, especially as I am based in France :)